### PR TITLE
fix for 'C1128: number of sections exceeded object file format limit'

### PR DESCRIPTION
**Pull request by ElderOrb**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2644/HighFidelity-Beta-PR11223-3473401bddd544a0a33e243b45796bef0a887bcf.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2644/HighFidelity-Beta-PR11223-3473401bddd544a0a33e243b45796bef0a887bcf.exe

**Comment by hyperlogic** [comment]
Can you move this set into the `if (WIN32)` section? -bigobj does not apply to other platforms.

File: interface/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 set(TARGET_NAME interface)
 project(${TARGET_NAME})
+set(CMAKE_CXX_FLAGS_DEBUG  "${CMAKE_CXX_FLAGS_DEBUG} -bigobj")
```
Comment (hyperlogic) [inline interface/CMakeLists.txt]: Can you move this set into the if (WIN32) section? -bigobj does not apply to other platforms.

Comment (ElderOrb) [inline interface/CMakeLists.txt]: Done.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2664/HighFidelity-Beta-PR11223-a1e0f31cd8ca0546195cc141653f4901f393f7e1.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2664/HighFidelity-Beta-PR11223-a1e0f31cd8ca0546195cc141653f4901f393f7e1.dmg

**Pull request merged**